core/src/test/resources/testnet.seednodes

```diff
@@ -0,0 +1,7 @@
+# nodeaddress.onion:port [(@owner)]
```

Comment:
They are all removed. Better not have them in the code otherwise people think they are running.